### ISSUE TITLE
text/plain attachments show \n instead of newlines in step view

### ISSUE THREAD

**Issue by dawn-minion** [wswebcreation/multiple-cucumber-html-reporter#98]
**Environment (please complete the following information):**
 - multiple-cucumber-html-reporter: 1.12.1
 - Node.js version: v12.9.1
 - Yarn version: 1.17.3
 - Platform name and version: Ubuntu 19.04
 - Cucucmber version: 5.1.0

**Config of multiple-cucumber-html-reporter**
```js
    reporter.generate({
        jsonDir: args.report,
        reportPath: args.output,
        customMetadata: true,
        displayDuration: true
    })
```

**Describe the bug**
Plain text attachments show "\n" instead of actual newlines. Example:
![demo](https://user-images.githubusercontent.com/35529725/63844484-9a1f7980-c988-11e9-8184-2424b17a046b.png)

**To Reproduce**
report.json to reproduce the above image:
```json
[
    {
        "keyword": "Feature",
        "name": "Demo Report.json for Bug",
        "line": 1,
        "id": "demo-report.json-for-bug",
        "tags": [],
        "uri": "features/myTest.feature",
        "elements": [
            {
                "id": "demo-report.json-for-bug;we-show-the-bug",
                "keyword": "Scenario",
                "line": 2,
                "name": "We show the bug",
                "type": "scenario",
                "steps": [{
                    "keyword": "Given ",
                    "line": 1,
                    "name": "we make an attachment with newlines",
                    "match": {
                        "location": "features/steps/myTest.js:1"
                    },
                    "result": {
                        "status": "passed"
                    },
                    "embeddings": [
                        {
                            "data": "Some text with newlines\nThe second line\nThe third line",
                            "mime_type": "text/plain"
                        }
                    ]
                }]
            }
        ]
    }
]
```

**Expected behavior**
I would expect actual newlines instead of \n - I'm using these attachments to debug steps that interact with files, so having all the file contents on a single line with \n makes it difficult to view them correctly. 

Here's how I would expect it to look:
![Schermafdruk van 2019-08-28 11-41-13](https://user-images.githubusercontent.com/35529725/63844579-c20edd00-c988-11e9-8e68-49362c48ce3b.png)

EDIT: Looking at the code that produces it, it seems this text box is just the result of a JSON.stringify(...) call on the `embeddings` array. Can we rework this so we show the actual text content instead of JSON data?

**Comment by wswebcreation** [comment]
This is fixed with version `1.13.0`, see https://github.com/wswebcreation/multiple-cucumber-html-reporter/releases/tag/v1.13.0

**Comment by dawn-minion** [comment]
Thank you @wswebcreation!
